integration_test/third_party_apps_data/test_config.yaml

```diff
@@ -7,6 +7,7 @@ per_application_overrides:
     platforms_to_skip: &common_skips
       - centos-7
       - centos-8
+      - rockylinux-8
```

Comment:
Note: the name of this platform on GCE is actually `rocky-linux-8`. I've fixed that in #377.